docs/move.md

```diff
@@ -130,11 +133,35 @@ This can now be achieved with the following procedure:
 
 9. Delete the bootstrap cluster
 
-**Important Note** The move process should be done when the BMHs are
-in a steady state. BMHs should not be moved while any operation is on-going i.e.
-BMH is in provisioning state. This will result in failure since the IP of the
-BMH might change after the move and the DHCP-leases from the management cluster
-are not moved to target cluster.
+## Impotant Notes
+
+The following requirements are essential for the move process to run
+successfully:
+
+1.
+The move process should be done when the BMHs are in a steady state. BMHs
+should not be moved while any operation is on-going i.e. BMH is in provisioning
+state. This will result in failure since the IP of the BMH might change after
+the move and the DHCP-leases from the management cluster are not moved to target
+cluster.
+
+2.
+Before the move process is initialized, it is important to delete the ironic
+pod/ironic containers. If ironic is deployed in cluster the deployment is named
+`metal3-ironic`, if it is deployed locally outside the cluster, the containers
+to be deleted are `ironic`, `ironic-inspector`, `ironic-endpoint-keepalived`,
+`ironic-dnsmasq`, `ironic-httpd` and `mariadb`. Deleting the ironic before move
+process makes sure that the old ironic does not interfere with the operations of
+the new ironic deployed in target cluster and also it makes sure that there is
+only one dnsmasq existent at any time.
```

Comment:
I would invert those two sentences to emphasize on why you should not have two ironic at once. Also, we should emphasize that the database of the first ironic instance is not cleaned when we move the BMHs because we move them without reconciling

Reply:
Done. Thanks